chore: added injury effects table

Doc-Sun opened the pull request:
<!--
For Work In Progress Pull Requests, please use the Draft PR feature.
See https://github.blog/2019-02-14-introducing-draft-pull-requests/ for further details.
-->

**Type**  
What type of pull request is this? (e.g., Bug fix, Feature, Refactor, etc.)

- [ ] Bug fix
- [x] Feature
- [ ] Refactor
- [ ] Other (please describe):

**Description**  
Added Missing tables from the beta rules that were missing that could be used for rolling or choosing.

**Related Issue**  
 Half closes Closes #183

**How Has This Been Tested?**  
Tested and built in foundry, exported .json and added to repo


**Checklist:**  
- [ ] I have commented on my code, particularly in hard-to-understand areas.
- [x] My changes do not introduce any new warnings or errors.
- [x] My PR does not contain any copyrighted works that I do not have permission to use.
- [x] I have tested my changes on Foundry VTT version: [12.331].

**Additional context**  
_Add any other context or information here that would be useful for reviewers._

MangoFVTT reviewed (changes requested):
Shardblade injuries are a Stormlight exclusive and should definitely be moved to a module (probably to the playtest materials for now). The other one can stay 👍 We'll need to think of styling for these as well I suspect, in order to be able to apply the effect directly from the chat cards

Doc-Sun commented:
> Shardblade injuries are a Stormlight exclusive and should definitely be moved to a module (probably to the playtest materials for now). The other one can stay 👍 We'll need to think of styling for these as well I suspect, in order to be able to apply the effect directly from the chat cards

Thanks. Removed the shardblades chart from this PR and renamed the PR

MangoFVTT commented on src/packs/tables/injury-effects.json:
```diff
@@ -0,0 +1,111 @@
+      "systemVersion": "0.2.1"
+    }
+  },
+  "_stats": {
+    "coreVersion": "12.331",
+    "systemId": "cosmere-rpg",
+    "systemVersion": "0.2.1",
+    "createdTime": 1732504183989,
+    "modifiedTime": 1732507522837,
+    "lastModifiedBy": "mlhqK8ft6P9hbaYE"
+  },
```
These stats should also be removed

Doc-Sun replied:
fixed in latest push

Doc-Sun commented on src/packs/tables/injury-effects.json:
```diff
@@ -0,0 +1,111 @@
+      ],
+      "drawn": false,
+      "_id": "gpJNDSRsoaWUM0T0",
+      "text": "<strong>Exausted(-1)</strong> Any Injury that lowers your overall stamina.",
```
fixed in latest push

Doc-Sun commented on src/packs/tables/injury-effects.json:
```diff
@@ -0,0 +1,111 @@
+      ],
+      "drawn": false,
+      "_id": "hp5hWMYB10llrOtH",
+      "text": "<strong>Exausted(-2)</strong> Any Injury that lowers your overall stamina.",
```
fixed in latest push

Doc-Sun commented on src/packs/tables/injury-effects.json:
```diff
@@ -0,0 +1,111 @@
+      "coreVersion": "12.331",
+      "systemVersion": "0.2.1"
+    }
+  },
```
fixed in latest push

MangoFVTT commented on src/packs/tables/injury-effects.json:
```diff
@@ -0,0 +1,97 @@
+  "displayRoll": true,
+  "folder": null,
+  "flags": {},
+  "_stats": {},
```
I believe _stats has to go completely, sorry should've clarified

Doc-Sun replied:
fixed and pushed

zithith commented on src/packs/tables/injury-effects.json:
```diff
@@ -0,0 +1,96 @@
+      ],
+      "drawn": false,
+      "_id": "Xsus0oW97bVGoZEW",
+      "text": "<strong>Supprised</strong> Overwhelmed due to the shock of an injury.",
```
Typo in Surprised

Doc-Sun replied:
fixed in push

zithith commented:
Would be kinda good to have the conditions somewhere as Items in the system so that these could have documentIds, but that's a thought for another day

Doc-Sun commented:
> Would be kinda good to have the conditions somewhere as Items in the system so that these could have documentIds, but that's a thought for another day

That would be nice (id really like them to be like the pf2e conditions that even show up in the top right of the screen with hover over text when placed on a actor token and that token is selected) but thats a little beyond me for today. This I can do
